build.sc

```diff
@@ -19,22 +19,24 @@ trait CompositeModule extends ScalaModule { outer =>
     for {
       root <- allSources()
       if os.exists(root.path)
-      path <- (if (os.isDir(root.path)) os.walk(root.path) else Seq(root.path))
+      path <- if (os.isDir(root.path)) os.walk(root.path) else Seq(root.path)
       if os
         .isFile(path) && ((path.ext == "scala" || path.ext == "java") && !isHiddenFile(path) && !isFromSubmodule(path))
     } yield PathRef(path)
   }
 
   trait Tests extends super.Tests with CompositeModule {
     def ivyDepsExtra: Agg[Dep] = Agg()
-    final def ivyDeps = ivyDepsExtra ++ testingLibrary
+    override final def ivyDeps = ivyDepsExtra ++ testingLibrary
```

Comment:
I believe `final` was here on purpose, so that children modules don't override and skip `testingLibrary`; children modules were supposed to use `ivyDepsExtra`; is there any reason you are changing it?